src/docs/firmware/updating-firmware.md

```diff
@@ -5,7 +5,7 @@ The script does not work on Windows. If you are using Windows, boot a live Linux
 :::
 
 ::: tip
-curl is not installed by default for Ubuntu / Ubuntu distros. To install it, run: `sudo apt update && sudo apt install -y curl`
+curl is not installed by default for Ubuntu or Ubuntu-based distros. To install it, run: `sudo apt update && sudo apt install -y curl`
```

Comment:
Why do we even have this note when we don't support Ubuntu

Reply:
We do recommend it for flashing only

Reply:
Should/could we change that? As it can cause confusion, and using fedora/debian is just as easy to use to flash nowadays.

The only reason Ubuntu is recommended is because the script was initially written back when Ubuntu was actually alright

Reply:
Not true, Ubuntu is the only iso that never needed an additional kernel param due to its lenient kernel

Reply:
Afaik Debian hasn’t either?

Reply:
Not that i know of. Canonical's "specialness" help us here